problems/2.functions.js

```diff
@@ -2,34 +2,44 @@
 * @param {string} name - The name of the user to greet
 * @returns {null}
 */
-function logGreeting() {
-
+function logGreeting(name) {
+  let name = null
+  return name
+  
 }
 
 /** Returns a custom greeting string in the form "Hello Avery!"
 * @param {string} name - The name of the user to greet
 * @returns {string} - The custom greeting
 */
-function greeting() {
+function greeting(name) {
 
 }
+console.log("Hello Avery")
 
 /** Adds a number to a global variable called globalCount
  * Access globalCount like this: global.globalCount
 * @param  {number} val - The number to add to globalCount
 * @returns {null}
 * @modifies {global.globalCount}
 */
+let globalCount = val
+
+
+
 function incrementGlobalCount() {
 
 }
```

Comment:
To update a variable you need to do reassignment and thus need the assignment operator (=). 
global.globalCount += val